packages/teams-js/src/private/messageChannels.ts

```diff
@@ -0,0 +1,59 @@
+import { requestPortFromParent } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { runtime } from '../public/runtime';
+
+/**
+ * @hidden
+ * Namespace to request message ports from the host application.
+ *
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export namespace messageChannels {
```

Comment:
This namespace should have an `isSupported` function like (almost) all of the other namespaces in TeamsJS. That function makes sure the host supports this capability. Developers can call it directly if they wish, but each function also checks the value of `isSupported` before sending messages to the host. (This is why @shrshindeMSFT asked you to call ensureInitialized, isSupported won't return the correct value before the sdk has been initialized).

Reply:
<img width="316" alt="image" src="https://github.com/OfficeDev/microsoft-teams-library-js/assets/56459261/10758477-1ee6-4019-a7c9-57700a40a6e8">

The pattern that I see in the other APIs is calling ensureInitialized twice - once directly, and then again as part of the isSupported check. This seems redundant?

Are we fine to just call isSupported with the assumption it is also calling ensureInitialized?